airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/manifest.yaml

```diff
@@ -0,0 +1,677 @@
+    retriever:
+      $ref: "#/definitions/semi_incremental_stream/retriever"
+      paginator:
+        $ref: "#/definitions/links_next_paginator_with_page_size"
+    $parameters:
+      name: "ticket_fields"
+      path: "ticket_fields"
+      cursor_field: "updated_at"
+      primary_key: "id"
+
+  ticket_forms_stream:
+    $ref: "#/definitions/semi_incremental_stream"
+    retriever:
+      $ref: "#/definitions/semi_incremental_stream/retriever"
+      paginator:
+        type: DefaultPaginator
+        pagination_strategy:
+          type: CursorPagination
+          cursor_value: '{{ response.get("next_page", {}) }}'
+          stop_condition: '{{ config.get("ignore_pagination") or not response.get("next_page", {}) }}'
+        page_token_option:
+          type: RequestPath
+    $parameters:
+      name: "ticket_forms"
+      path: "ticket_forms"
+      cursor_field: "updated_at"
+      primary_key: "id"
+
+  ticket_metric_events_stream:
+    $ref: "#/definitions/base_incremental_stream"
+    retriever:
+      $ref: "#/definitions/retriever"
+      ignore_stream_slicer_parameters_on_paginated_requests: true
+      paginator:
+        $ref: "#/definitions/links_next_paginator_with_page_size"
+    $parameters:
+      name: "ticket_metric_events"
+      path: "incremental/ticket_metric_events"
+      cursor_field: "time"
+      cursor_filter: "start_time"
+      primary_key: "id"
+
+  ticket_metrics_stream:
+    $ref: "#/definitions/semi_incremental_stream"
+    retriever:
+      $ref: "#/definitions/semi_incremental_stream/retriever"
+      ignore_stream_slicer_parameters_on_paginated_requests: true
+      paginator:
+        $ref: "#/definitions/links_next_paginator_with_page_size"
+    $parameters:
+      name: "ticket_metrics"
+      path: "ticket_metrics"
+      cursor_field: "updated_at"
+      primary_key: "id"
+
+  ticket_skips_stream:
+    $ref: "#/definitions/semi_incremental_stream"
+    retriever:
+      $ref: "#/definitions/semi_incremental_stream/retriever"
+      requester:
+        $ref: "#/definitions/retriever/requester"
+        request_parameters:
+          sort_order: "desc"
+      paginator:
+        $ref: "#/definitions/links_next_paginator"
+    $parameters:
+      name: "ticket_skips"
+      path: "skips.json"
+      data_path: "skips"
+      cursor_field: "updated_at"
+      primary_key: "id"
+
+  tickets_stream:
+    $ref: "#/definitions/base_incremental_stream"
+    retriever:
+      $ref: "#/definitions/retriever"
+      ignore_stream_slicer_parameters_on_paginated_requests: true
+      paginator:
+        $ref: "#/definitions/after_url_paginator"
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_datetime_formats:
+        - "%s"
+      datetime_format: "%s"
+      cursor_field: "{{ parameters.get('cursor_field') }}"
+      start_datetime:
+        datetime: "{{ timestamp(config.get('start_date')) | int or day_delta(-3650, '%Y-%m-%dT%H:%M:%SZ') }}"
```

Comment:
It was 2 years ago by default in previous version, why it was changed to 10 years?